style.css

```
@@ -0,0 +1,174 @@
+*{
+    margin:0;
+}
+body{
+    box-sizing: border-box;
+    font-family: Poppins, 'Noto Sans KR', sans-serif;
+}
+
+header{
+    height: 70px;
+    max-width: 1920px;
+     width: 100%;
+     margin: 80px auto;
+     margin-top: 0;
+     margin-bottom: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.head-block{
+    width: 100%;
+    max-width: 1080px;
+    margin-left: 200px;
+    margin-right: 200px;
```

요런건 단축 속성을 사용해주면 코드를 줄여줄수있겠죠?
```suggestion
    margin: 0 200px;
```